torch/testing/_internal/common_methods_invocations.py

```diff
@@ -11251,8 +11267,8 @@ def ref_pairwise_distance(input1, input2):
         ref=lambda x, threshold, value: np.where(x > threshold, x, value).astype(x.dtype),
         dtypes=all_types_and(torch.bfloat16),
         dtypesIfCUDA=all_types_and(torch.float16, torch.bfloat16),
-        supports_forward_ad=False,
```

Comment:
Do you know why these were specified?

Reply:
Hmm not sure, maybe the author just wanted to be explicit that forward isn't implemented?